pkgs/by-name/li/librenms-agent/package.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchFromGitHub, nixosTests }:
+
+stdenv.mkDerivation {
+  pname = "librenms-agent";
+  version = "1.2.6b5";
+
+  src = fetchFromGitHub {
+    owner = "librenms";
+    repo = "librenms-agent";
+    # upstream provides no git tags
+    rev = "f6d6ff5b88bd47e738754fe72c4a4e5eb4e78d08";
+    hash = "sha256-b62xkwJ09BQe/XRXZqnOr5JUpLjLIqYRETr/wTim/h4=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D -m 0750 check_mk_agent -t $out/bin/
+
+    runHook postInstall
+  '';
+
+  passthru.tests = nixosTests.librensm-agent;
```

Comment:
```
error: attribute 'librensm-agent' missing

       at /dev/shm/nixpkgs-review/pr-271427/nixpkgs/pkgs/by-name/li/librenms-agent/package.nix:23:20:

           22|
           23|   passthru.tests = nixosTests.librensm-agent;
             |                    ^
           24|
       Did you mean librenms-agent?
```

```suggestion
  passthru.tests = nixosTests.librenms.agent;
```

Reply:
ofborg should've raised an error on this :/

Reply:
whoops, thats a typo on my end

`passthru.tests = nixosTests.librenms-agent;`

nixpkgs-review does not build `.passthru.tests`, i tried so manually because i noticed that ofborg didn't build it

Reply:
@pbsds , done, please re-review.